src/e2e/java/teammates/e2e/pageobjects/AppPage.java

```diff
@@ -64,6 +64,9 @@
     @FindBy(linkText = "Profile")
     private WebElement studentProfileTab;
 
+    @FindBy(linkText = "View team")
```

Comment:
This element can't be in `AppPage`; it's very specific to the student home page, if I recall correctly.